<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>465</width>
    <height>313</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Service Options</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>10</y>
     <width>461</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="ServerTab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Server Port:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="srvportSpin">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="srvconnectSpin">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>67</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>65</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max Connections:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseButton">
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>99</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>106</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbpathEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="DatabaseTab">
    <attribute name="title">
     <string>Database</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>61</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>User name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>95</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>126</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbusrEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>61</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbpasswdEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>94</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbnameEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>127</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Server Address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="srvaddrEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>27</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Other</string>
    </attribute>
    <widget class="QTableView" name="pcTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="hardTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>401</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addDistItemBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>390</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rmvDistItemBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>390</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="disttypeGroup">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>401</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>Distribute Type</string>
     </property>
     <widget class="QRadioButton" name="pcRadioBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>PC</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="hardRadioBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Hard</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="noneRadioBtn">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>84</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupDisNum">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Distribute Count</string>
     </property>
     <widget class="QSpinBox" name="spinDistOrgNum">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinDistNum">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelDistOrgNum">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>23</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Original Num</string>
      </property>
     </widget>
     <widget class="QLabel" name="labeDistNum">
      <property name="geometry">
       <rect>
        <x>194</x>
        <y>23</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>490</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="serviceButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Service</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
